.editorconfig

```diff
@@ -34,3 +34,9 @@ indent_size = 2
 # Match perl/python/shell scripts, set indent width of four
 [*.{pl,py,sh}]
 indent_size = 4
+
+# Disable for auto-generated hackage2nix file
+[pkgs/development/haskell-modules/hackage-packages.nix]
```

Comment:
While you are at it, maybe also check if we need to add `pkgs/development/node-packages/node-packages-v*.nix`

Reply:
Apart from not having final newlines they seem fine.

Some of the generated json files (`node-packages-*.json`, `recipes-archive-melpa.json`) will probably need to be excluded if we end up adding `json`.

Reply:
Added `node-packages-v*.nix` and `gemset.nix`.

I guess we're probably going to end up doing this for a lot of the generated files.